docs/maintenance.rst

```diff
@@ -52,7 +52,7 @@ Steps for creating a new Zope release
 
 - Upload the tagged release to PyPI::
 
-    python2.7 setup.py egg_info -RDb '' sdist register upload
+    python2.7 setup.py egg_info -RDb '' sdist bdist_wheel register upload
```

Comment:
FYI, the ``register`` keyword is obsolete. You can do a straight ``upload`` without ``register``.